README.md

```diff
@@ -74,16 +74,18 @@ In addition, the Silk.NET working group help drive larger user-facing changes pr
 <h1 align="center">Building from source</h1>
 
 Prerequisites
-- **Must**: .NET 6 SDK
-- **Should**: [NUKE](https://nuke.build) (build system). Install using `dotnet tool install Nuke.GlobalTool --global`
-- **Should**: Android, iOS, and MAUI .NET 6 workloads (use `dotnet workload install android ios maccatalyst maui` to install them)
-- **Should**: Android SDK version 30 with NDK tools installed. On Windows, for best results this should be installed into `C:\ProgramData\Android\android-sdk`.
-- **Could**: Java JDK (for gradle)
-- **Could**: Visual Studio 2022 Community version 17.0 or later
+- .NET 7 SDK
+- Android, iOS, and Mac Catalyst workloads (use `dotnet workload install android ios maccatalyst` to install them)
+    - On Linux, `ios` and `maccatalyst` should be omitted as they are not available
+- Android SDK version 30 with NDK tools installed
+    - On Windows, for best results this should be installed into `C:\ProgramData\Android\android-sdk`
+- Java 11 JDK
+- Visual Studio 2022 Community version 17.0 or later (optional)
```

Comment:
Why did you remove the must/should/could?

Reply:
Because the only one that's actually "could" now is Visual Studio. All the other stuff is mandatory to do a successful `./build.sh`.

Reply:
ok